index.html

```diff
@@ -25,12 +27,185 @@ <h2 class="padding">I’m Youssef Glad to see you!</h2>
       </p>
       <a class="padding" href="#">LET’S CONNECT</a>
       <div class="social">
-        <img src="/assets/127.svg" alt="" />
-        <img class="sc-item" src="/assets/125.svg" alt="" />
-        <img class="sc-item" src="/assets/124.svg" alt="" />
-        <img class="sc-item" src="/assets/126.svg" alt="" />
-        <img class="sc-item" src="/assets/123.svg" alt="" />
+        <img src="./assets/127.svg" alt="" />
+        <img class="sc-item" src="./assets/125.svg" alt="" />
+        <img class="sc-item" src="./assets/124.svg" alt="" />
+        <img class="sc-item" src="./assets/126.svg" alt="" />
+        <img class="sc-item" src="./assets/123.svg" alt="" />
       </div>
     </section>
+    <!-- Works Section -->
+    <section class="works">
+      <div class="cards">
+        <!-- First Card -->
+        <div class="card">
+          <img class="card-img" src="./assets/Snapshoot1.png" alt="" />
+          <div class="card-details">
+            <div class="card-item-mg">
+              <span class="card-title">Tonic</span>
```

Comment:
- [ ] Please make sure that your project title (e.g., "Tonic") uses a header tag `hx`. :pray:

Reply:
fixed